files/es/learn/common_questions/thinking_before_coding/index.html

```diff
@@ -128,38 +128,38 @@ <h3 id="¿Qué_hay_que_hacer_y_en_qué_orden_para_alcanzar_mis_metas">¿Qué hay
     <ol>
      <li>Escribe algunos artículos para iniciar la discusión</li>
      <li>Define cómo deben lucir los artículos</li>
-     <li>Publica los artículos en el sitio web (¿Cómo hacer esto?)</li>
+     <li>Publica esos artículos en el sitio web (¿Cómo hacer esto?)</li>
     </ol>
    </td>
   </tr>
   <tr>
-   <td style="vertical-align: top;">Conocer otros músicos</td>
+   <td style="vertical-align: top;">Conocer a otros músicos</td>
    <td>
     <ol>
-     <li>Provee modos para que las personas te contacten (¿Email? ¿Facebook? ¿Teléfono? ¿Correo?)</li>
+     <li>Provee mecanismos para que las personas te contacten (¿Email? ¿Facebook? ¿Teléfono? ¿Correo?)</li>
      <li>Define cómo las personas encontrarán esos canales de contacto desde tu sitio web</li>
     </ol>
    </td>
   </tr>
   <tr>
-   <td style="vertical-align: top;">Venta de golosinas</td>
+   <td style="vertical-align: top;">Vender golosinas</td>
    <td>
     <ol>
-     <li>Crea las golosinas</li>
+     <li>Produce las golosinas</li>
      <li>Almacena las golosinas</li>
-     <li>Encuentra un modo de manejar el envío</li>
-     <li>Encuentra una manera de manejar el pago</li>
-     <li>Hacer un mecanismo en tu sitio para que la gente haga pedidos</li>
+     <li>Encuentra un modo de manejar los envíos</li>
+     <li>Encuentra una manera de manejar los pagos</li>
+     <li>Implementa un mecanismo en tu sitio para que la gente haga pedidos</li>
```

Comment:
* implementar es el paso siguiente de hacer

```suggestion
     <li>Hacer un mecanismo en tu sitio para que la gente haga pedidos</li>
```

Reply:
@Graywolf9, en este punto, en esta lista, una sola cosa (nuevamente, acato lo que decidas, pero...): toda la lista precedente debería estar en un miso tiempo verbal. O va en infinitivo -todo-, o va en segunda persona del imperativo. Por una cuestión de concordancia. Yo preferí, y lo confieso, cambiar lo menos posible, esto es el imperativo a infinitivo, pero si preferís el infinitivo, deberíamos cambiar todo el resto de la lista a ese mismo tiempo verbal.
UNA CUESTIÓN ADICIONAL: como "Haz", el imperativo, me "sonaba" mal, puse "Implementa", pero bueno, en esto también quedo a lo que indiques.

Reply:
Suena lógico, tienes razón, lo marco como resuelto, gracias